netaddr.go

```diff
@@ -13,7 +13,7 @@
 package netaddr // import "inet.af/netaddr"
 
 import (
-	"bytes"
+	"encoding/binary"
```

Comment:
We could avoid this import pretty easily if we felt the need. We're invoking pretty trivial functions that get inlined, nothing complex.

Reply:
This import doesn't worry me.